backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -276,6 +279,7 @@ def _delete_zaak(self):
 
     def process_deletion(self) -> None:
         self.processing_status = InternalStatus.processing
+        self.extra_zaak_data = get_zaak_metadata(self.zaak)
```

Comment:
Are we sure we want to save the meta data within OAB, don't we want OZ as single source of truth for the remaining data?

Reply:
Mhmm :thinking: But how would we save the data in OZ? I don't think that there is any field on a zaak that would be suitable for storing this information, and updating a document for every zaak that is deleted would be very cumbersome for large lists :thinking: